yrt-pet/executables/recon/Reconstruct.cpp

```diff
@@ -384,7 +389,22 @@ int main(int argc, char** argv)
 		osem->attenuationImageForForwardProjection = invivoAttImg.get();
 
 		// Save steps
-		osem->setSaveSteps(saveSteps, out_fname);
+		Util::RangeList ranges;
+		if (saveSteps > 0)
+		{
+			for (int it = 0; it < numIterations; it += saveSteps)
```

Comment:
If `saveSteps == 1`, it would be simpler (and faster) to add only one range spanning from `0` to `numIterations - 1`.

Reply:
done